cfme/tests/automate/test_automate_manual.py

```diff
@@ -1227,3 +1147,77 @@ def test_miq_stop_abort_with_state_machines():
         1441353
     """
     pass
+
+
+@pytest.mark.tier(3)
+def test_validate_method_uri_data():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: automate
+        testSteps:
+            1. Create domain > namespace > class
+            2. Click to 'Add a new method' with type 'uri'
+            3. Enter value in 'Data' field
+        expectedResults:
+            1.
+            2.
+            3. 'Validate' button is getting enabled
+
+    Bugzilla:
+         1684417
+    """
+    pass
+
+
+@pytest.mark.tier(3)
+def test_user_requester_for_lifecycle_provision():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: automate
+        testSteps:
+            1. Tested a series of service provisions and default lifecycle provisions with different
+               users (Jay only did default, usera only did generic service provision)
+            2. Added logging to the validate_request method for both Service and Infrastructure
+               namespaces
+            3. Viewed the automation.log
+        expectedResults:
+            1.
+            2.
+            3. Request user should be correct
```

Comment:
Also, it's better to define what it means "correct". Something like `The same user is shown in the automation.log as in the request`